tools/scylla-nodetool.cc

```diff
@@ -4302,6 +4317,34 @@ For more information, see: {}
                 removenode_operation
             }
         },
+        {
+                {
+                    "excludenode",
+                    "Mark nodes as permanently down (excluded).",
+    fmt::format(R"(
+The cluster will no longer attempt to contact excluded nodes, which unblocks
```

Comment:
nit: s/, which unblocks/.  This can be used to unblock/

Reply:
Also, I'd mention that the nodes are fenced off and that they are not allowed to rejoin the cluster,
even if they somehow recover.